charmtools/build/builder.py

```diff
@@ -328,6 +329,16 @@ def plan_layers(self, layers, output_files):
                                        layer=layer,
                                        next_config=next_config,
                                        output_files=output_files))
+        if self.wheelhouse_overrides:
+            existing_tactic = output_files.get('wheelhouse.txt')
+            output_files['wheelhouse.txt'] = WheelhouseTactic(
+                self.wheelhouse_overrides,
+                self.target,
+                layers["layers"][-1],
+                next_config,
+            )
+            if existing_tactic is not None:
+                output_files['wheelhouse.txt'].combine(existing_tactic)
```

Comment:
Should we combine or just be an overwrite?

Reply:
Nevermind, thinking about it combine makes more sense

Reply:
Combine merges the files.  In fact, we end up with both versions of the files in the wheelhouse, which is also something that can happen with multiple layers referencing the same dep.  It would be great to be able to resolve that up front, but that would require reimplementing much of pip's dependency resolution, so I'm disinclined to even try.  It does generally do the right thing at the end, but it may be something we need to address eventually.